spring-cloud-kubernetes-fabric8-autoconfig/src/main/java/org/springframework/cloud/kubernetes/fabric8/Fabric8AutoConfiguration.java

```diff
@@ -98,10 +98,6 @@ public Config kubernetesClientConfig(KubernetesClientProperties kubernetesClient
 				.withProxyPassword(or(kubernetesClientProperties.getProxyPassword(), base.getProxyPassword()))
 				.withNoProxy(or(kubernetesClientProperties.getNoProxy(), base.getNoProxy())).build();
 
```

Comment:
since we have introduced [namespace resolution concept](https://docs.spring.io/spring-cloud-kubernetes/docs/current/reference/html/#namespace-resolution), this message is a bit inaccurate and can bring some confusion to users.  

Instead, I propose to drop it, and bring a bit of a clarification in a different part of the documentation.